Automattic/wp-calypso
# Help Center: Use correct Odie bot version

Related to p1732099777076489-slack-C07D72S1135

## Proposed Changes

- Use Odie 14.0.0 for the experiment
- Default version otherwise

## Testing Instructions



* Test with a flag
** Start a new chat with Odie and talk to it.
** Search the call to get your chat in the network tab `/wpcom/v2/odie/chat/wpcom-support-chat/[chat id]` and check that `bot_version` in the response is 14.0.0
* Do the same without a flag, but the response should be 13.1.0
